pom.xml

```diff
@@ -5,28 +5,79 @@
 
     <modelVersion>4.0.0</modelVersion>
 
+    <parent>
+        <groupId>io.airlift</groupId>
+        <artifactId>airbase</artifactId>
+        <version>144</version>
+    </parent>
+
     <groupId>io.trino.gateway</groupId>
-    <artifactId>trinogateway-parent</artifactId>
-    <name>trinogateway-parent</name>
+    <artifactId>trino-gateway-parent</artifactId>
     <packaging>pom</packaging>
     <version>3-SNAPSHOT</version>
 
+    <name>trino-gateway-parent</name>
+    <description>A smart load balancer and query router for Trino</description>
+    <url>https://github.com/trinodb/trino-gateway</url>
+    <inceptionYear>2018</inceptionYear>
+
+    <licenses>
+        <license>
+            <name>Apache License 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+    <developers>
+        <developer>
+            <name>Trino Gateway contributors</name>
+            <url>https://github.com/trinodb/trino-gateway/graphs/contributors</url>
+        </developer>
+        <developer>
+            <name>Lyft Presto Gateway contributors</name>
+            <url>https://github.com/lyft/presto-gateway/graphs/contributors</url>
+        </developer>
+    </developers>
+    <scm>
+        <connection>scm:git:git://github.com/trinodb/trino-gateway.git</connection>
+        <tag>HEAD</tag>
+        <url>https://github.com/trinodb/trino-gateway</url>
+    </scm>
+
     <properties>
-        <maven.compiler.source>17</maven.compiler.source>
-        <maven.compiler.target>17</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
+        <project.build.targetJdk>17</project.build.targetJdk>
+        <air.java.version>17</air.java.version>
+        <air.check.skip-checkstyle>true</air.check.skip-checkstyle>
+        <air.check.skip-enforcer>true</air.check.skip-enforcer>
+        <air.check.skip-license>true</air.check.skip-license>
+        <air.check.skip-dependency>true</air.check.skip-dependency>
+        <air.check.skip-duplicate-finder>true</air.check.skip-duplicate-finder>
+        <air.check.skip-modernizer>true</air.check.skip-modernizer>
+        <air.check.fail-spotbugs>false</air.check.fail-spotbugs>
+        <air.check.skip-pmd>true</air.check.skip-pmd>
+
+        <!-- overriding version from airbase -->
+        <dep.jackson.version>2.10.1</dep.jackson.version>
+        <dep.logback.version>1.2.12</dep.logback.version>
```

Comment:
it's declared by airbase. drop

Reply:
This is a necessary override .. without it the project does not work/build ..we will update deps after first release